docs/2_circuit.md

```diff
@@ -6,7 +6,7 @@ First, build a basic circuit by connecting the RGB LED and the PIR sensor to the
 
 <span class="notes">**Note:** Before you start, put the SD card in the SD card reader on your development board.</span>
 
-Mbed Cloud requires an external storage to store the identity of the device, and to store firmware updates. This external storage does not necessarily have to be an SD card (which is too expensive to put in a production device), you can also use internal flash or external SPI flash. To modify the storage layer, you'll need to modify the bootloader (see 'Section 8 - Firmware updates') and the application (see `mbed_app.json` in the `connected-lights-cloud` application later in this tutorial); which is not covered in this tutorial. More information is in the [Mbed Cloud docs](https://cloud.mbed.com/docs/).
+Device Management requires an external storage to store the identity of the device, and to store firmware updates. This external storage does not necessarily have to be an SD card (which is too expensive to put in a production device), you can also use internal flash or external SPI flash. To modify the storage layer, you'll need to modify the bootloader (see 'Section 8 - Firmware updates') and the application (see `mbed_app.json` in the `connected-lights-cloud` application later in this tutorial); which is not covered in this tutorial. More information is in the [Pelion Device Management docs](https://cloud.mbed.com/docs/).
```

Comment:
Might need "Pelion Device Management", because of device management also being a concept, but I think this demand comes from Pelion.

Reply:
Yeah, changed it so the first mention actually has the full name.